include/devicetree/spi.h

```diff
@@ -116,6 +116,46 @@ extern "C" {
  */
 #define DT_SPI_DEV_HAS_CS_GPIOS(spi_dev) DT_SPI_HAS_CS_GPIOS(DT_BUS(spi_dev))
 
+/**
+ * @brief Get a SPI device's chip select devicetree specification
+ *
+ * Example devicetree fragment:
+ *
+ * @code{.devicetree}
+ *     gpio1: gpio@... { ... };
+ *
+ *     gpio2: gpio@... { ... };
+ *
+ *     spi@... {
+ *             compatible = "vnd,spi";
+ *             cs-gpios = <&gpio1 10 GPIO_ACTIVE_LOW>,
+ *                        <&gpio2 20 GPIO_ACTIVE_LOW>;
+ *
+ *             a: spi-dev-a@0 {
+ *                     reg = <0>;
+ *             };
+ *
+ *             b: spi-dev-b@1 {
+ *                     reg = <1>;
+ *             };
+ *     };
+ * @endcode
+ *
+ * Example usage:
+ *
+ * @code{.c}
+ *     DT_SPI_DEV_CS_GPIOS_DT_SPEC_GET(DT_NODELABEL(a)) \
+ *           // { DT_NODELABEL(a), 10, GPIO_ACTIVE_LOW }
+ *     DT_SPI_DEV_CS_GPIOS_DT_SPEC_GET(DT_NODELABEL(b)) \
+ *           // { DT_NODELABEL(b), 20, GPIO_ACTIVE_LOW }
```

Comment:
nit: `DEVICE_DT_GET(DT_NODELABEL(a/b))`

Reply:
Do you mean
<pre>
// { DEVICE_DT_GET(DT_NODELABEL(b)), 20, GPIO_ACTIVE_LOW }
</pre>
and
<pre>
// { DEVICE_DT_GET(DT_NODELABEL(b)), 20, GPIO_ACTIVE_LOW }
</pre> ?

Reply:
correct

Reply:
Done :)